src/pages/[spaceId]/index.tsx

```diff
@@ -42,6 +57,21 @@ const getLastPosts = async (commentIdsByPostId: string[][]) => {
   }
   return lastPosts
 }
+async function getPostsData(postIds: string[]) {
+  const subsocialApi = await getSubsocialApi()
+
+  const promises = postIds.map((postId) => {
+    return subsocialApi.blockchain.getReplyIdsByPostId(postId)
+  })
+  const postsPromise = getPostsFromCache(postIds)
+
+  const commentIdsByPostId = await Promise.all(promises)
+  const posts = await postsPromise
```

Comment:
Why do you not put `postsPromise` into `Promise.all`?

Reply:
Because postsPromise is a Promise<PostData[]>
and I didn't wrap both of them into single Promise.all to separate the variables